docs: goose x Hacktoberfest 2025 Blog (#4855)
block/goose: documentation/blog/2025-09-26-hacktoberfest-2025/index.md

---
title: "goose is Celebrating Hacktoberfest 2025!"
description: Explore goose prizes you can earn and contributions you can make during Hacktoberfest 2025
authors: 
    - tania
---

![blog banner](hacktoberfest2025.png)

> October is around the corner, which means spooky season is upon us, and with that crispy fall air to make your "gooses" bump...it's finally time for Hacktoberfest 2025! The goose team is beyond excited to celebrate with you all for the first time this year. Let's get into how you can participate, and what prizes you can win. 👀

Reviewer: ```suggestion
October is around the corner, which means spooky season is upon us, and with that crispy fall air that gives you "goose" bumps...it's finally time for Hacktoberfest 2025! The goose team is beyond excited to celebrate with you all for the first time this year. Let's get into how you can participate, and what prizes you can win. 👀
```

- If your PR is marked as `spam`, `invalid`, or needs more input from you, you need to re-evaluate your contribution. This is to ensure it counts towards Hacktoberfest, and that your PR aligns with the repository's goals.
- Outside of guidelines of a given issue, always make sure your contributions adhere to our code of conduct.

## 🏆 Leaderboard & Prizes
Every hacktoberfest PR and contribution will earn you points on our leaderboard. The Top 20 Contributors with the most points by the end of the month will win special prizes! The leaderboard will auto-update with each contribution, so here's a breakdown of how our ranking system works:

| Weight | Points Awarded | Description |
| -------- | -------- | -------- |
| 🐭 Small     | 5 points     | For smaller issues that take limited time to complete and/or don't require any product knowledge.     |
| 🐰 Medium     | 10 points     | For average issues that take additional time to complete and/or require some product knowledge.     |
| 🐂 Large     | 15 points     | For meaty issues that take a significant amount of time to complete and/or possibly require deep product knowledge.     |

> As thanks for every contribution we receive this year, the team is offering exclusive gift cards for the fresh new goose swag shop along with free LLM credits for our Top 20:

Reviewer: ```suggestion
As a special thank you, the team is offering exclusive gift cards for the fresh new goose swag shop along with free LLM credits for our Top 20 contributors:
```
